pkg/dashboard/templates/dashboard.gohtml

```diff
@@ -38,6 +38,9 @@
 
   <div class="layoutSidebar__main">
     <main class="verticalRhythm --rhythm-3">
+      <a class="banner" href="https://www.fairwinds.com/goldilocks-users-insights?utm_source=goldilocks&utm_medium=ad&utm_campaign=goldilocksad" target="_blank">
+        <img src="/static/images/insights.png" alt="Apply K8s right sizing and best practices across multiple clusters and teams">
+      </a>
```

Comment:
* Wrap in aside for screen readers to be able to skip over easier
* Add additional context to alt text so that it makes more sense standalone and indicates new window
* add rel=noreferrer for security - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#security_and_privacy

```suggestion
      <aside class="banner">
        <a href="https://www.fairwinds.com/goldilocks-users-insights?utm_source=goldilocks&utm_medium=ad&utm_campaign=goldilocksad" target="_blank" rel="noreferrer">
          <img src="/static/images/insights.png" alt="Apply K8s right sizing and best practices across multiple clusters and teams with Fairwinds Insights. Try Demo. (Opens in new window.)">
        </a>
      </aside>
```